src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -90,6 +94,10 @@ protected override bool IsAsyncSupportingFunctionSyntax(SyntaxNode node)
                     // in Task<>.
                     newReturnType = taskOfTType.Construct(methodSymbol.ReturnType).GenerateTypeSyntax();
                 }
+                else
+                {
+                    newReturnType = newReturnType.WithAdditionalAnnotations(Formatter.Annotation);
```

Comment:
@zaytsev-victor Can you take a look and see if this would produce a cleaner result? Let me know what you decide.